server/src/storage/s3.rs

```diff
@@ -474,12 +476,12 @@ impl ObjectStorage for S3 {
         let file_format = ParquetFormat::default().with_enable_pruning(Some(true));
         let listing_options = ListingOptions {
             file_extension: ".parquet".to_string(),
-            file_sort_order: Vec::default(),
+            file_sort_order: vec![vec![col(DEFAULT_TIMESTAMP_KEY).sort(true, false)]],
             infinite_source: false,
             format: Arc::new(file_format),
             table_partition_cols: vec![],
             collect_stat: true,
-            target_partitions: 1,
+            target_partitions: 32,
```

Comment:
This section is repeated for local and s3 mode. Can we move it to the common abstraction?

Reply:
Yes, this needs refactoring